modules/apps/headless/headless-admin-content/headless-admin-content-impl/src/main/java/com/liferay/headless/admin/content/internal/resource/v1_0/StructuredContentResourceImpl.java

```diff
@@ -139,6 +144,29 @@ public EntityModel getEntityModel(MultivaluedMap multivaluedMap) {
 			});
 	}
 
+	@Override
+	public Page<StructuredContent> getStructuredContentsVersionsPage(
+			@NotNull Long structuredContentId)
```

Comment:
Better removing the @NotNull annotation from here. As far as I know it shouldn't appear here, only in the Base...ResourceImpl